modules/backing-up-red-hat-quay-standalone.adoc

```diff
@@ -0,0 +1,108 @@
+:_content-type: PROCEDURE
+[[backing-up-red-hat-quay-standalone]]
+= Backing up {productname} on standalone deployments 
+
+This procedure describes how to create a backup of {productname} on standalone deployments. 
+
+.Prerequisites
+
+.Procedure 
+
+. Create a temporary backup directory, for example, `quay-backup`: 
++
+[source,terminal]
+----
+$ mkdir /tmp/quay-backup
+----
+
+. Change into a directory that bind-mounts to `/conf/stack` inside of the container by running the following command: 
++
+[source,terminal]
+----
+$ cd /opt/quay
+----
+
+. Compress the contents of your {productname} deployment into an archive in the `quay-backup` directory by entering the following command: 
++
+[source,terminal]
+----
+$ tar cvf /tmp/quay-backup/quay-backup.tar.gz *
+----
++
+Example output:
++
+[source,terminal]
+----
+config.yaml
+config.yaml.bak
+extra_ca_certs/
+extra_ca_certs/ca.crt
+ssl.cert
+ssl.key
+----
+
```

Comment:
We should probably advise the user to also back up the container run command (and thus the container specification) with something like this:

```bash
$ podman inspect quay-app | jq -r '.[0].Config.CreateCommand | .[]' | paste -s -d ' ' -

/usr/bin/podman run --name quay-app -v /etc/quay-install/quay-config:/quay-registry/conf/stack:Z -v /etc/quay-install/quay-storage:/datastorage:Z registry.redhat.io/quay/quay-rhel8:v3.7.5
```